docs/source/training/socialisation/run_basic_inasafe.rst

```diff
@@ -99,7 +99,7 @@ the source of the data.
 |                              |               | the Australian Government.                                                      |
 |                              |               | `See more at... <http://inasafe.org/en/training/socialisation/datasets.html>`__ |
 +------------------------------+---------------+---------------------------------------------------------------------------------+
-| Java_population_WGS84        | WorldPop      | High resolution, modelled data for human population distributions.              |
+| Jakarta_population_WGS84        | WorldPop      | High resolution, modelled data for human population distributions.              |
```

Comment:
Lovely comments @adissadis - so nice to see such clear indications of the issue.